apps/prairielearn/src/schemas/schemas/infoAssessment.json

```diff
@@ -163,41 +146,45 @@
     }
   },
   "definitions": {
-    "groupRole": {
-      "description": "A custom role for use in group assessments that allows control over certain permissions.",
-      "type": "object",
-      "properties": {
-        "name": {
-          "description": "The group role's name (i.e. Manager, Reflector, Recorder).",
-          "type": "string"
-        },
-        "minimum": {
-          "description": "The minimum number of users that should be in this role in a group.",
-          "type": "number",
-          "default": 0
-        },
-        "maximum": {
-          "description": "The maximum number of users that should be in this role in a group.",
-          "type": "number"
-        },
-        "canAssignRoles": {
-          "description": "Whether users with this role can assign other users' group roles.",
-          "type": "boolean",
-          "default": false
-        }
-      }
+    "PointsJsonSchema": {
```

Comment:
Nit: reorder these definitions to make this easier to review?

Reply:
I believe they have now been reordered.